app/js/components/manageApps/ManageApps.jsx

```diff
@@ -362,8 +362,11 @@ export default class ManageApps extends React.Component {
     });
   }
 
-  handleUserClick(name) {
-    toastr.info(`Sorry, there is no open web app for ${name}`);
+  handleUserClick(appType, name) {
+    appType === "module"?
```

Comment:
What is the appType for an owa? Is it also "module"?

Reply:
For owa's appType would be OWA. Made it such that the message is specific to when OWA's that are not installed informs the user to install the OWA, otherwise the user is informed that the OWA does not exist